### DISCUSSION TITLE
Recent versions are not published to Cocoapods

### DISCUSSION THREAD

**Discussion by karthikgs7** [pointfreeco/swift-issue-reporting#55] [Q&A] [answered]
The latest version which is available in cocoa pods is 0.1.1. Is it possible to publish the newer one there?

**Comment by stephencelis** [comment, marked as answer]
Hi @karthikgs7!

We actually don't currently provide CocoaPods support for any of our libraries, and I don't think we ever supported CocoaPods for XCTest Dynamic Overlay. It looks like CocoaPods has sunset a lot of its functionality, like seeing who owns a particular pod spec, so it's hard to verify, but I think it may have been submitted by someone else.

It should be possible for you to maintain your own pod spec for XCTest Dynamic Overlay if you can't use SPM for some reason and/or want to use CocoaPods and need a more recent version. I also did a quick search on GitHub and found a few folks maintaining their own pod specs:

https://github.com/search?utf8=✓&q=xctestdynamicoverlay+podspec&type=code

I'm going to convert this to a discussion, since it's not a bug with the library.

**Comment by jshier** [comment]
It looks like this podspec sources https://github.com/adimobile/xctest-dynamic-overlay.git rather than the PointFree version, so you may want to start with that org. Or like Stephen said, you can create your own podspec to use yourself.
